client/src/components/AppBarRight.tsx

```diff
@@ -34,20 +35,27 @@ export class AppBarRight extends React.Component<AppBarRightProps, PopoverState>
             open: true,
             anchorEl: event.currentTarget,
         });
-    };
+    }
 
     handleRequestClose = () => {
         this.setState({
             open: false,
         });
-    };
+    }
 
     handleLogoutClose = () => {
         this.setState({
             open: false,
         });
         this.props.onLogoutClick ? this.props.onLogoutClick() : '';
-    };
+    }
+
+    handleGradesClose = () => {
```

Comment:
This is the same function as handleLogoutClose and presumably always would be. Any chance we can redo the semantic and just handleMenuClose?